it/hubs_privacy_notice.md

```diff
@@ -59,6 +59,16 @@ __Provider di servizi di ricerca:__ Puoi cercare immagini, GIF e modelli 3D da c
 * [YouTube](https://policies.google.com/privacy)
 * [Twitch](https://www.twitch.tv/p/legal/privacy-policy/)
 
-__Twitter:__ Se connetti Twitter a Hubs, puoi twittare e condividere immagini 2D dalle stanze di Hubs. Ogni contenuto twittato verrà condiviso con Twitter e pubblicato sulla piattaforma Twitter. Per ulteriori informazioni, consulta l'[Informativa sulla privacy di Twitter](https://twitter.com/en/privacy).
+__X (ex Twitter):__ Se connetti X a Hubs, puoi twittare e condividere immagini 2D dalle stanze di Hubs. Ogni contenuto twittato verrà condiviso con X e pubblicato sulla piattaforma X. Per ulteriori informazioni, consulta l'[Informativa sulla privacy di X](https://twitter.com/privacy).
 
 __Discord:__ Se connetti Discord a Hubs, memorizziamo i token di accesso e gli ID dei server e dei canali che sono stati connessi. Sincronizzeremo i messaggi di chat, le modifiche alle stanze, gli oggetti 2D e 3D che crei e l'accesso o l'uscita con il canale Discord collegato. Hubs non registra i messaggi sincronizzati. Per ulteriori informazioni, consulta l'[Informativa sulla privacy di Discord](https://discordapp.com/privacy).
+
+## Altre cose da sapere
+
+[Qui](https://support.mozilla.org/kb/firefox-accounts-managing-account-data) puoi trovare ulteriori informazioni sulla gestione dei dati del tuo account Mozilla. Inoltre, molte delle informazioni che memorizziamo sui nostri utenti di account Mozilla sono facilmente accessibili [dal tuo account](https://accounts.firefox.com/signin), dove puoi anche aggiornare le [impostazioni di condivisione dei dati](https://accounts.firefox.com/settings/). Per inoltrare richieste relative ai tuoi dati personali, contattaci sul nostro [Portale per la richiesta di accesso ai dati personali](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
+
+Per qualsiasi altra domanda in merito ai tuoi dati personali alle nostre pratiche di privacy, puoi contattarci all'indirizzo compliance@mozilla.com.
```

Comment:
The translation is missing the `or` (`o` in Italian), so it doesn't make a lot of sense. I'm going to fix it and merge.

Reply:
@AdrianUrsea Please report back to the translator (also made another small change to clarify)
https://github.com/mozilla/legal-docs/pull/1954/commits/39bab0e9e2e72e2c4aa250e1c056b0aabfccf85c

Reply:
I will document translation related issues and report back